packages/ibm-products/src/components/Tearsheet/TearsheetShell.tsx

```diff
@@ -270,6 +272,13 @@ export const TearsheetShell = React.forwardRef(
     );
     const modalRefValue = modalRef.current;
 
+    // Function to strip html tags out of a string.
+    const stripTags = useCallback(
```

Comment:
We wont need useCallback in React 18 so no need to add this for now

Reply:
Will just add more cleanup later on

Reply:
Thank you!